WcaOnRails/config/initializers/translations.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+if defined?(Rails::Server) # See: https://stackoverflow.com/a/44441168
+  Rails.configuration.after_initialize do
+    modification_timestamp = Timestamp.find_or_create_by!(name: 'en_translation_modification')
+    latest_modification_date = DateTime.parse(`git log -1 --format='%ai' #{Rails.root.to_s}/config/locales/en.yml`)
+    if modification_timestamp.date != latest_modification_date
```

Comment:
This is a bit more tricky! Consider the following scenario: someone works on a PR1 and modifies `en.yml` on Monday; after that someone else makes a new PR2 on Tuesday and deploys it immediately; the `en_translation_modification` timestamp points to Tuesday; then PR1 gets merged&deployed, but since `en.yml` was modified on Monday as a part of PR1, translators are not notified... Not really sure how to deal with this one.